packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -3422,30 +3462,32 @@ export default abstract class SqlIntegration
         }
       )
       ${
-        quantileMetric === "event"
+        quantileMetric === "event" // TODO(sql): put quantiles in their own query
```

Comment:
checking if this is a before-merge or a future work TODO.

Reply:
future work.